code/modules/reagents/reagent_containers/food/snacks.dm

```diff
@@ -1570,6 +1570,42 @@
 	reagents.add_reagent(CARAMEL, 3)
 	bitesize = 3
 
+/obj/item/weapon/reagent_containers/food/snacks/explosive_pie
+	name = "banana cream pie"
```

Comment:
instead of these 3 lines and the code in New(), why not just make this a child of banana cream pie so it inherits the name, desc, icon, and reagents?

Reply:
He removed the pie inheritance so it can't spawn on a wizard pie spells.

Reply:
That makes sense enough I suppose

Reply:
I'd actually like for it to spawn on pie-wiz. Friendly wizard turned explosive murderer. (((Fun))) for the whole family